docs/source/reference/numpysupported.rst

```diff
@@ -427,8 +427,12 @@ Polynomials
 
 The following polynomial functions are supported:
 * :func:`numpy.polynomial.polynomial.polyadd()`
+* :func:`numpy.polynomial.polynomial.polydiv()`
+* :func:`numpy.polynomial.polynomial.polyint()` (only the 2 first arguments)
 * :func:`numpy.polynomial.polynomial.polymul()`
 * :func:`numpy.polynomial.polynomial.polysub()`
+* :func:`numpy.polynomial.polynomial.polyval()` (only the 2 first arguments)
```

Comment:
The problem of supporting the `tensor` argument in `polyval` is because it changes the shape of the resulting array?

Reply:
Hi @guilhermeleobas, you are correct. The ``tensor`` argument changes the output shape. I suppose it can be supported as a literal boolean.

Reply:
Could you add the case when this variable is a literal boolean